Documentation/dev-guide/interacting_v3.md

```diff
@@ -267,14 +314,29 @@ bar
 PUT
 foo
 bar_new
+```
 
+```bash
 # watch for changes on key `foo` since revision 3
 $ etcdctl watch --rev=3 foo
 PUT
 foo
 bar_new
 ```
 
+Here is an example to watch only from the last historical change:
+
+```bash
```

Comment:
@xiang90 can you tell me what part you have confusion in? it will help me to re-frame and re-structure it better.
Thank you!